packages/gatsby-source-vtex/tsconfig.json

```diff
@@ -1,8 +1,35 @@
 {
-  "extends": "../../tsconfig.json",
+  // see https://www.typescriptlang.org/tsconfig to better understand tsconfigs
```

Comment:
I don't understand why remove extends?

Reply:
I've just fallback to tsdx's standard definition. But I can merge with extends